packages/itwin/property-grid/src/test/hooks/UseInstanceSelection.test.tsx

```diff
@@ -312,4 +313,72 @@ describe("useInstanceSelection", () => {
       expect(result.current.ancestorsNavigationProps.canNavigateUp).to.be.true;
     });
   });
+
+  describe("feature usage reporting", () => {
+    const initialProps: InstanceSelectionProps = {
+      imodel,
+    };
+
+    it("reports when navigates to parent", async () => {
+      const onFeatureUsedSpy = sinon.spy();
+      selectionManager.getSelection.returns(new KeySet([childKey]));
+
+      const wrapper = ({ children }: { children: React.ReactNode }) => (
+        <TelemetryContextProvider onFeatureUsed={onFeatureUsedSpy}>{children}</TelemetryContextProvider>
+      );
+      const { result } = renderHook(useInstanceSelection, { initialProps, wrapper });
+
+      await waitFor(() => {
+        expect(result.current.selectedKeys[0].id).to.be.eq(childKey.id);
+        expect(result.current.ancestorsNavigationProps.canNavigateUp).to.be.true;
+      });
+
+      await act(async () => result.current.ancestorsNavigationProps.navigateUp());
+
+      await waitFor(() => {
+        expect(result.current.selectedKeys[0].id).to.be.eq(parentKey.id);
+      });
+
+      expect(onFeatureUsedSpy).to.be.calledOnceWith("ancestor-navigation");
+    });
+
+    it("reports when navigates down initial instance", async () => {
+      const onFeatureUsedSpy = sinon.spy();
+      selectionManager.getSelection.returns(new KeySet([childKey]));
+
+      const wrapper = ({ children }: { children: React.ReactNode }) => (
+        <TelemetryContextProvider onFeatureUsed={onFeatureUsedSpy}>{children}</TelemetryContextProvider>
+      );
+      const { result } = renderHook(useInstanceSelection, { initialProps, wrapper });
+
+      await waitFor(() => {
+        expect(result.current.selectedKeys[0].id).to.be.eq(childKey.id);
+        expect(result.current.ancestorsNavigationProps.canNavigateUp).to.be.true;
+      });
+
+      await act(async () => result.current.ancestorsNavigationProps.navigateUp());
+
+      await waitFor(() => {
+        expect(result.current.selectedKeys[0].id).to.be.eq(parentKey.id);
+        expect(result.current.ancestorsNavigationProps.canNavigateDown).to.be.true;
+      });
+
+      expect(selectionManager.replaceSelection).to.be.calledOnceWith(
```

Comment:
Does not like this is relevant to the test.

Reply:
You are right, removed.